src/modules/pos/validator/components/StakeForm/StakeForm.js

```diff
@@ -153,11 +167,46 @@ const StakeForm = ({ t, stakes, account, isStakingTxPending, nextStep, history,
     posToken
   );
   const showEmptyState = !changedStakes.length || isStakingTxPending;
+  const onConfirm = async (formProps, transactionJSON, selectedPriority, fees) => {
+    if (!showEmptyState && moduleCommandSchemas && !isLoading) {
```

Comment:
the `!isLoading` condition seems redundant here? Isnt the confirm button disabled when `isLoading`? 
Now it reads like

> if isLoading (but this function is somehow anyway is called)
> go to history.push(routes.validators.path);